### DISCUSSION TITLE
theres no documentation on the website about .exe thingy

### DISCUSSION THREAD

**Discussion by Ryzelz** [TomSchimansky/CustomTkinter#1611] [Q&A] [answered]
trying to package my files into .exe using pyinstaller but the .json file is conflicting

![image](https://github.com/TomSchimansky/CustomTkinter/assets/100330023/3f1ad8f8-3f43-4e68-b997-6d61c859be58)

**Comment by avalon60** [comment]
Are you talking of Pyinstaller? If so, it's not actually part of CustomTkinter. I wouldn't be surprised if Tom had removed it from the docs, as it serves as a distraction from core CustomTkinter issues.

**Reply by Ryzelz** [reply]
i see, so any work around on this problem?

**Reply by avalon60** [reply, marked as answer]
> i see, so any work around on this problem?

It's a long time since I used Pyinstaller, as it's somewhat limited, but you could look at https://github.com/TomSchimansky/CustomTkinter/discussions/939

This still remains, but it just boils down to a couple of tips:
https://customtkinter.tomschimansky.com/documentation/packaging
